configuration-inventory/configuration-inventory-requirements.md

```diff
@@ -4,6 +4,10 @@ description: This article lists the requirements necessary to successfully deplo
 
 # Configuration Inventory Requirements
 
+{% hint style="warning" %}
+**Before you continue**, check if the Configuration Inventory feature is included in your [subscription plan](https://www.syskit.com/products/point/pricing/).
+{% endhint %}
+
```

Comment:
rekli smo da mičemo ovaj tip note-a
svuda bi trebao biti ovaj tip kao što je u tipa audit logs collection gdje kažemo od kojeg je plana feature/report/nešto dostupno